game/selenium_tests/base.py

```
@@ -47,13 +50,14 @@ def setUpClass(cls):
+            cls.driver = webdriver.Chrome(
+                service=Service(ChromeDriverManager().install()),
+                options=chrome_options
+            )
             log_ok("Chrome WebDriver initialized")
         except Exception as e:
-            log_fail(f"Failed to initialize Chrome WebDriver: {e}")
-            raise RuntimeError(
-                f"Failed to initialize Chrome WebDriver: {e}"
-            ) from e
+            log_warn(f"Skipping Selenium tests: Chrome WebDriver not found ({e})")
+            raise unittest.SkipTest("Chrome WebDriver not available")
```

Ruff (0.15.12)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)